src/tracks/InstrumentTrack.cpp

```diff
@@ -1638,12 +1655,26 @@ void InstrumentTrackWindow::modelChanged()
 		m_pitchRangeLabel->hide();
 	}
 
+	if (m_track->instrument() && m_track->instrument()->flags().testFlag(Instrument::IsMidiBased))
+	{
+/*		m_miscView->microtunerGroupBox()->hide();
```

Comment:
I don't want those comments in this PR, but if you still want to include them, you can comment out the whole `if`\~`else` block.

Reply:
I can delete it, I was just trying to make my life easier by making the future conflicts more obvious / less likely to miss.